src/app/shared/constants.ts

```diff
@@ -33,7 +33,8 @@ export enum METADATA_VALIDATION_STATES {
   Draft = 'Draft',
   Validating = 'Validating',
   Valid = 'Valid',
-  Invalid = 'Invalid'
+  Invalid = 'Invalid',
+  GraphInvalid = 'Causes Invalid Graph'
```

Comment:
'Causes' is confusing here.

Reply:
It is intentional. It does not actually contain a graph validation error because it is just one entity, it causes an error